incubator/letsencrypt-setup/templates/_helpers.tpl

```diff
@@ -60,6 +60,9 @@ solvers:
     route53:
       hostedZoneID: {{ .hostedZoneID | default "" }}
       region: {{ .region }}
+      {{- if .role }}
+      role: {{ .role }}
+      {{- end }} {{- /* endif .role */ -}}
```

Comment:
Is the second thing just a comment? I haven't seen this pattern before, and it seems a bit unnecessary to me

Reply:
Ha I hadn't either, I was just matching the pattern used in the rest of the file. It does seem pretty verbose.

Reply:
Huh, TIL. Works for me. We're probably going to be abandoning this chart in the near-ish future anyway.